demisto_sdk/commands/content_graph/interface/graph.py

```diff
@@ -47,13 +52,46 @@ def commit(self) -> Optional[str]:
             return self.metadata.get("commit")
         return None
 
+    @property
+    def content_parser_commit(self) -> Optional[str]:
+        if self.metadata:
+            return self.metadata.get("content_parser_commit")
+        return None
+
+    @property
+    def schema(self) -> Optional[dict]:
+        try:
+            return get_file(self.import_path / self.SCHEMA_FILE_NAME)
+        except FileNotFoundError:
+            return None
+
     def dump_metadata(self) -> None:
         """Adds metadata to the graph."""
+        # I want to save the latest commit from this page: https://github.com/demisto/demisto-sdk/commits/master/demisto_sdk/commands/content_graph/parsers using API
+        parser_commit = self._get_content_parser_commit_hash()
         metadata = {
             "commit": GitUtil().get_current_commit_hash(),
+            "content_parser_commit": parser_commit,
```

Comment:
commit and content_parser_commit is confusing, please think of more indicative names

Reply:
how about `content_parser_latest_commit`?

Reply:
sababa

Reply:
maybe also change commit to content_repo_latest_commit

Reply:
This is not the latest commit from content, this is the content commit when the graph is created/updated